ui/plugin-lifecycle/README.md

```diff
@@ -0,0 +1,209 @@
+# VCD UI Plugin Seed #
```

Comment:
Either make this a valid README for the lifecycle plugin, or remove it.

Reply:
Removed in initial commit.